app/views/accounts/_auth_apps.html.erb

```diff
@@ -19,10 +19,12 @@
 </div>
 <% if current_user.auth_app_configurations.count < IdentityConfig.store.max_auth_apps_per_account %>
   <div class="grid-col-auto">
-    <%= link_to(
-          prefix_with_plus(t('account.index.auth_app_add')),
```

Comment:
I checked out this branch, and it looks like after these changes there's only one use of `prefix_with_plus` and it's on another account page partial, should we clean it up too and remove the method?

```
identity-idp:stevegsa-account-buttons2> git grep prefix_with_plus
app/helpers/text_helper.rb:  def prefix_with_plus(str)
app/views/accounts/actions/_regenerate_backup_codes.html.erb:  <%= prefix_with_plus(t('forms.backup_code.regenerate')) %>
```

Reply:
yes